include/rocksdb/c.h

```diff
@@ -1765,6 +1771,10 @@ extern ROCKSDB_LIBRARY_API uint64_t
 rocksdb_approximate_memory_usage_get_cache_total(
     rocksdb_memory_usage_t* memory_usage);
 
+extern ROCKSDB_LIBRARY_API void rocksdb_options_set_dump_malloc_stats(rocksdb_options_t*, int);
```

Comment:
Could you change the type of the second parameter to `unsigned char`? That's what we generally use as a stand-in for `bool` in the C API. Note that this applies to the next method as well.

Reply:
done